source/Calamari/Scripts/Octopus.Features.IISWebSite_BeforePostDeploy.ps1

```diff
@@ -713,10 +713,16 @@ if ($deployAsWebSite)
 					}
 				}
 			} else { # SNI off so we will have created against the ip
-				$existing = & netsh http show sslcert ipport="$($ipAddress):$port"
-				if ($LastExitCode -eq 0) {
-					Write-Host ("Removing unused SSL certificate binding: $($ipAddress):$port")				
+				# check if there are any other bindings to the same IP:Port so that
+				# we don't remove an ssl cert that's used by any other sites on the server
+				$existing = Get-WebBinding -IPAddress $ipAddress -Port $port
```

Comment:
What if the binding on that ip:port combo that you find with this command is using HTTP and has no cert attached to it. This command would return a false positive. You would still want to remove the orphaned SSL cert binding. I think it makes sense to add the flag `-Protocol https` to make sure it actually has a cert attached to it.